tutorials/get-started/get-started3.mdx

```diff
@@ -261,7 +263,144 @@ Your luos device have been successfully mounted into a "device" object:
       ┃  ╰> Unknown             blinker             4    ┃
      >┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
 ```
+</TabItem>
+<TabItem value="No board" label="No board">
+
+As we saw in Part 1, Luos engine allows you to define services and use them together on one program. What really sets Luos apart and makes it special is that you can also make services work together on separated programs.
+
+In Part 1, you have downloaded or cloned the Get started code folder in your computer. We will use this code to demonstrate how Luos engine works using a network.
+To create a link between multiple programs on your computer Luos will use WebSockets toexchange information instead of a serial communication between electronic boards.
+
+To be able to do that, you need to run a broker that will share any transmitted message with all other programs.
+
+This broker is a simple python script, to make it run you need to install simple_websocket_server:
+
+```
+pip install simple_websocket_server==0.4.2
+```
+
+Then you can run the broker script on `Get_started/No-Board/`:
+
+```bash
+python broker.py
+```
+
+:::info
+By default the broker will use localhost IP address.
+If you want to experiment it using multiple computers you can configure your IP and port:
+```bash
+python broker.py --ip 'YOUR_LOCAL_IP' -p 8000
+```
+:::
+
```

Comment:
Add a part here for GCC installation for windows users only. It could be something like that:
"Windows users must install a specific 64 bits version of GCC to compile the no board projects.
For example, the current version has been tested with MinGW-w64 (MSVCRT runtime). This library can be downloaded on  https://winlibs.com/"